docs/assets/css/header.less

```diff
@@ -29,88 +23,101 @@
     vertical-align: middle;
 }
 
-.respond-to-min( @bp-xs-max, {
+// Tablet and mobile size.
+.respond-to-max( @bp-sm-max, {
 
     .site-header {
-        padding: unit( 30px / @base-font-size-px, em ) 0;
-        border-bottom: 4px solid @green;
+        background-color: @gray-5;
+        display: table;
+        width: 100%;
+        padding: 0;
+    }
 
-        .content-l_col {
-            vertical-align: middle;
-        }
+    .title {
+        display: table-header-group;
+        margin-top: 0;
     }
 
-    .title_heading {
-        font-size: unit( 16px / @base-font-size-px, em );
-        margin: 0;
+    .title-link {
+        margin-top: 15px;
+        margin-left: 15px;
+        display: block;
     }
 
-    .global-nav {
-        text-align: right;
+    #search-form {
+        margin-top: 15px;
+        margin-bottom: 15px;
+        padding-left: 15px;
     }
 
+
     .global-nav_list {
-        display: inline-block;
-        font-size: unit( 12px / @base-font-size-px, em );
+        width: 100%;
     }
 
     .nav-list-item {
-        .u-inline-block;
-        margin-right: 20px;
-        a {
-            &:active,
-            &.active,
-            &:visited {
-                border-bottom: 1px solid @link-underline;
-                color: @link-text;
-            }
-            &:hover,
-            &:focus {
-                border-bottom: 1px solid @link-underline-active;
-                color: @link-text-active;
-            }
+        &:first-child a {
+            border-top: 1px solid @gray-20;
         }
-    }
+        a {
+            display: block;
+            border-bottom: 1px solid @gray-20;
+            padding: 1em;
 
-    .nav-list-item:last-child {
-        margin-right: 0px;
+        }
     }
 } );
 
+// Desktop size.
 .respond-to-min( @bp-sm-max, {
-    .title_heading {
-        font-size: 1.625em;
-    }
-    .global-nav_list {
-        font-size: unit( 16px / @base-font-size-px, em );
+
+    #search-form {
+        margin: 0;
+        float: right;
     }
-} );
 
-.respond-to-max( @bp-xs-max, {
+    .site-header-wrapper {
+        padding-bottom: unit( 30px / @base-font-size-px, em );
+        border-bottom: 4px solid @green;
+    }
 
     .site-header {
-        background-color: @gray-5;
+        max-width: 1180px;
```

Comment:
Should we be using breakpoint or grid variables for values like this? Or is this `max-width` super specific to this element?

Reply:
So we could tie it to [`@grid_wrapper-width`](https://github.com/cfpb/design-system/blob/f4613641062df12454f3101e09232a7b94096f0e/packages/cfpb-grid/src/cfpb-grid.less#L15) and do the math to downgrade it to the values here. I dunno, what do you think? On the one hand I'd err on the side of not coupling the website so much with the DS variables. I think the DS website should be bare minimum independent CSS as much as possible so that the website for displaying the components doesn't break when the components are broken. On the other hand, these values are here to align them with the content area, which does use the DS layout classes and variables.

Reply:
> I think the DS website should be bare minimum independent CSS as much as possible so that the website for displaying the components doesn't break when the components are broken.

I def agree with this statement so I vote for whatever is most congruent with it.